### ISSUE TITLE
Accordions do not follow ARIA pattern

### ISSUE THREAD

**Issue by syarwick23** [stephenravenscraft/loyola-template-4.0#121]
Just noticed this in SI: 

Accordions don't follow the [ARIA pattern](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/) prescribed, which causes issues with roles not in required elements. Fixing the Accordion will fix both SI issues below.

[SiteImprove Issue (Role not inside the required context)](https://my2.siteimprove.com/Accessibility/59595/NextGen/Issue/1?ruleId=42&pageSegments=&issueKind=1&exceptTags=1,2&conformance=&siteTargetIssueKinds=1,2&wcagVersion=20&siteTarget.wcagVersion=20&siteTarget.conformanceLevels=0,1&siteTarget.issueKinds=1,2)
[SiteImprove Issue (Container element is empty)](https://my2.siteimprove.com/Accessibility/59595/NextGen/Issue/1?ruleId=68&pageSegments=&issueKind=1&exceptTags=1,2&conformance=&siteTargetIssueKinds=1,2&wcagVersion=20&siteTarget.wcagVersion=20&siteTarget.conformanceLevels=0,1&siteTarget.issueKinds=1,2)

**Comment by stephenravenscraft** [comment]
Added role="button" instead of type="button". Question @syarwick23 - What container is empty? I'll keep reviewing. If I find it, you'll know from my update here.

**Comment by syarwick23** [comment]
There's not real 'empty' container, it just thinks it's empty because it doesn't have the correct children. The `role=tablist` should have a direct child of `role=tabpanel`, but there's a `div` in between

<img width="1355" height="421" alt="Image" src="https://github.com/user-attachments/assets/f4c774f9-fdac-4ff9-b4e6-21a473f6b434" />
